### DISCUSSION TITLE
Welcome to id42!

### DISCUSSION THREAD

**Discussion by faermanj** [CaravanaCloud/id42#1] [Announcements]
We are building the next-gen open-source delivery app, with many innovations for riders, commerces and customers!
What would YOU like to see on this app?

**Comment by michelpereira** [comment]
Hi, there.
Please have a look at:

https://github.com/appjusto

**Reply by faermanj** [reply]
Muito legal, valeu, nao conhecia!

**Comment by paulloeduardo** [comment]
Show , Obrigado pelo projeto !

**Reply by faermanj** [reply]
TMJ!

**Comment by brpeters** [comment]
Hello everybody, i am noob on this world of community share git-ops and open-source projects, excuse me my english, but i'am learning to, every day, everey single step a day, i want to learn how i can collaborate on this project and with these masters on twitch TV, Jeferson and Julio. Cya.

**Reply by faermanj** [reply]
no worries my friend, thank you and welcome!!!

**Comment by ericafolli** [comment]
Hi everyone! Excited to join this project! 🚀
